Rakefile

```diff
@@ -55,9 +55,11 @@ end
 desc "Test Datadog Process agent"
 task :test do
   cmd = "go list ./... | grep -v vendor | xargs go test"
+  tags = ["secrets"]
```

Comment:
this build tag does not seem to exist anywhere

Reply:
I thought it was required but it seems to work without it @sunhay any thought ?

Reply:
https://github.com/DataDog/datadog-agent/blob/e98a1266a754ef3fa33973cf7189c7ef1ecb1a46/pkg/secrets/fetch_secret.go

Reply:
Nevermind I had an outdated local version 👍